.github/workflows/ci-mlir.yml

```diff
@@ -46,6 +46,7 @@ jobs:
         # Change directory so that xdsl-opt can be found during installation.
         cd xdsl
         pip install -r requirements.txt
+        pip install --upgrade coverage
```

Comment:
I'm surprised we need this, isn't coverage in requirements.txt? If not, let's add it, so everyone can run the CI locally